projects/shared-models/notification.model.ts

```diff
@@ -5,9 +5,13 @@ import { IUser } from './user.model';
 import { IUserMessage } from './user_message.model';
 
 export interface INotification {
+  created_at: Date;
   entity: IUser | ILab | ICommunityBuild | IUserMessage;
-  sender: IUser;
+  entity_type: string;
   id: string;
   notification_message: string;
+  notification_message_type: string;
```

Comment:
Let's convert this to `enum` instead of `string`

Reply:
done